docs/vscode_install.md

```diff
@@ -16,12 +16,12 @@ The following is a way to integrate VS-Preview into VSCode.
     1. Select the "Add Item" button.
     1. Enter `*.vpy` for the "Item".
     1. Enter `python` for the "Value".
-    1. Select the "OK" button.
+    1. Select the "Okay" button.
```

Comment:
It's OK, not Okay
![it_is_ok_not_okay](https://user-images.githubusercontent.com/41454651/173222467-80aa3ba1-9860-4f89-a2fd-10b02a000994.png)